ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -357,6 +358,27 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 				Description: "Allow outbound connections to public destinations",
 			},
 
+			EnableSecureByDefaultFlag: {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Enable Secure-by-default on existing clusters (note: can be used on existing clusters)",
+				ValidateFunc: func(i interface{}, s string) (warnings []string, errors []error) {
```

Comment:
Why we are checking if its type bool or value can be true or false since we defined it as bool type will not it implictly take only true/false

Reply:
you are right, the first check is not needed. I'll remove it